conjur/api/client.py

```diff
@@ -90,12 +87,20 @@ def __init__(self,
                     if field_value:
                         on_disk_config[field_name] = field_value
                 loaded_config = on_disk_config
-                # pylint: disable=logging-fstring-interpolation
+                # Raise exception if the client was initialized in insecure mode
+                # but a follow-up request is run without the insecure flag
+                if ssl_verify is True and loaded_config['ca_bundle'] == '':
```

Comment:
when ssl_verify is true ?

Reply:
ssl_verify is the mode the user runs the CLI (T/F). They can run the CLI in --insecure mode then ssl_verify = False. Otherwise, it will be True